mpisppy/utils/sputils.py

```diff
@@ -984,7 +984,8 @@ def nonant_cost_coeffs(s):
         if id(var) in s._mpisppy_data.varid_to_nonant_index:
             raise RuntimeError(
                 "Found nonlinear variables in the objective function. "
```

Comment:
Change "Found nonlinear..." to "A call to nonant_cost_coefficient found nonlinear" (even though the user could figure that out from the stack trace, I don't want to give the impression that nonlinear is not OK in general")